x-pack/plugins/observability_solution/apm/common/alerting/config/apm_alerting_feature_ids.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ValidFeatureId } from '@kbn/rule-data-utils';
+
+export const apmAlertingFeatureIds: ValidFeatureId[] = [
+  'logs',
```

Comment:
Why not AlertConsumer... instead of hardcoded strings?

Reply:
I just wanted to prevent mixing concepts and this is safe because these are typed with `ValidFeatureId`, so adding a random string isn’t possible. 

However, if the `AlertConsumers` values are modified for any reason, this would trigger a type check error, requiring manual updates, so let's just use the const values 👍🏻 

https://github.com/elastic/kibana/pull/202025/commits/6c69928675a28d5d845a49ba1bfb6adb13acd08c